config/jobs/kubernetes-sigs/aws-ebs-csi-driver/aws-ebs-csi-driver-presubmits.yaml

```diff
@@ -2,10 +2,10 @@ presubmits:
   kubernetes-sigs/aws-ebs-csi-driver:
   - name: pull-aws-ebs-csi-driver-test-e2e-external-eks-windows
     cluster: eks-prow-build-cluster
-    always_run: true
     decorate: true
     skip_branches:
     - gh-pages
+    skip_if_only_changed: "^docs/.*$|^examples/.*$|^README.md$|^CHANGELOG.md$"
```

Comment:
np: you can drop the `.*$` in the docs and example entries, and that seems to be more common in `skip_if_only_changed` configs on this repo

more important non-np: suggest replacing `^README.md$|^CHANGELOG.md$` with `\.md$` to skip any edit to (only) markdown files rather than hardcoding specific files.

Reply:
+1 this will also catch CONTRIBUTING.md

Reply:
Might not be able to do `\.md$` due to `found unknown escape character`, will settle for `.md$` as other examples do.